corona-warn-app/cwa-app-android
# Show high risk card when having a positive test result (EXPOSUREAPP-12115)

[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12115)

How to test:

- Scan a positive test.
- Change your risk status to high -> the risk card should show up on the home screen.
- Change you risk status to low -> the risk status doesn't show up for the threshold period (currently 7 days). 
- Time travel 7+ days into the future -> the risk card should show up regardless of the risk status.
